modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -16357,6 +16357,7 @@ the-given-external-reference-code-belongs-to-another-account=The given external
 the-given-external-reference-code-belongs-to-another-account-group=The given external reference code belongs to another account group.
 the-google-client-id-is-invalid=Please enter a valid Google client ID.
 the-google-client-secret-is-invalid=Please enter a valid Google client secret.
+the-google-drive-file-was-deleted=The Google Drive file was deleted during the checkout process. Will check the file back with the saved value.
```

Comment:
What about "The Google Drive file was not found during the checkout process. The contents previous to the checkout have been restored.". Does this accurately reflect what is happening here?

Reply:
Or alternatively "The Google Drive file was missing ...".